src/Layout/redist/targets/Directory.Build.targets

```diff
@@ -10,16 +10,16 @@
 
   <ImportGroup Condition="'$(GenerateSdkBundleOnly)' != 'true'">
     <Import Project="BundledSdks.targets" />
-    <Import Project="PublishDotnetWatch.targets" />
+    <Import Project="BundledDotnetTools.targets" />
+    <Import Project="GenerateBundledVersions.targets" />
     <Import Project="GenerateLayout.targets" />
-    <Import Project="OverlaySdkOnLKG.targets" Condition="'$(DotNetBuild)' != 'true'" />
 
     <Import Project="BundledTemplates.targets" />
-    <Import Project="BundledDotnetTools.targets" />
-    <Import Project="GenerateBundledVersions.targets" />
     <Import Project="Crossgen.targets" />
     <Import Project="GenerateInstallerLayout.targets" />
     <Import Project="GenerateArchives.targets" />
+
+    <Import Project="OverlaySdkOnLKG.targets" />
```

Comment:
Was removing the DotNetBuild condition intentional (I haven't had a chance to refresh my memory on what that was for)?

Reply:
Yes this removes one difference between the VMR product build and the sdk isolated repo build. The condition was necessary before but with these changes isn't anymore.